staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -406,6 +415,7 @@ func (w *watchCache) UpdateResourceVersion(resourceVersion string) {
 		w.Lock()
 		defer w.Unlock()
 		w.resourceVersion = rv
+		defer w.cond.Broadcast()
```

Comment:
Probably missing something obvious, but why defer as a last statement of the function call?

Reply:
Removed.